kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, model_part, params, file_header):
 
         default_settings = KratosMultiphysics.Parameters('''{
             "output_file_name"  : "",
+            "save_in_folder"    : true,
+            "output_folder"     : "TimeBasedAsciiResults",
```

Comment:
I would make only one parameter `output_folder_name` with a default of `""`
If the user specifies a name then it writes to a folder, otherwise it doesn't
What do you think?